spec/rubocop/cop/style/word_array_spec.rb

```diff
@@ -559,7 +559,7 @@
       RUBY
     end
 
-    it "doesn't fail with `encoding: binary" do
+    it "doesn't fail with `encoding: binary`" do
```

Comment:
This change is unrelated to this PR, right? It might be better to open it as a separate PR.

Reply:
Sure, that was just an occasional style fix.